WebJobHost/Web.config

```diff
@@ -1,76 +1,83 @@
-    <add key="docdb.PrimaryAuthorizationKey" value="config:DocDBKey" />
-    <add key="docdb.DatabaseId" value="DevMgmtDB" />
-    <add key="docdb.DocumentCollectionId" value="DevMgmtCollection" />
-    <add key="iotHub.HostName" value="config:IotHubName" />
-    <add key="iotHub.ConnectionString" value="config:IotHubConnectionString" />
-    <add key="device.StorageConnectionString" value="config:ServiceStoreAccountConnectionString" />
-    <add key="device.TableName" value="config:DeviceTableName" />
-    <add key="DeviceRulesStoreContainerName" value="devicerules" />
-    <add key="DeviceRulesTableName" value="DeviceRulesNormalizedTable" />
-    <add key="ActionMappingStoreContainerName" value="actionmappings" />
-    <add key="ActionMappingStoreBlobName" value="mappings.json" />
-    <add key="AsaRefDataRulesBlobName" value="devicerules.json" />
-    <add key="SolutionName" value="Remote Monitoring Solution" />
-    <add key="TelemetryStoreContainerName" value="devicetelemetry" />
-    <add key="TelemetryDataPrefix" value="devicetelemetry/" />
-    <add key="TelemetrySummaryPrefix" value="devicetelemetry-summary/" />
-    <add key="AlertsStoreContainerName" value="rulesoutput" />
-    <add key="DeviceAlertsDataPrefix" value="" />
-    <add key="MapApiQueryKey" value="config:MapApiQueryKey" />
+    <add key="ida.AADTenant" value="config:AADTenant"/>
+    <add key="ida.AADAudience" value="config:AADAudience"/>
+    <add key="ida.FederationMetadataAddress" value="config:AADMetadataAddress"/>
+    <add key="ida.FederationRealm" value="config:AADRealm"/>
+    <add key="docdb.EndpointUrl" value="config:DocDbEndPoint"/>
+    <add key="docdb.PrimaryAuthorizationKey" value="config:DocDBKey"/>
+    <add key="docdb.DatabaseId" value="DevMgmtDB"/>
+    <add key="docdb.DocumentCollectionId" value="DevMgmtCollection"/>
+    <add key="iotHub.HostName" value="config:IotHubName"/>
+    <add key="iotHub.ConnectionString" value="config:IotHubConnectionString"/>
+    <add key="device.StorageConnectionString" value="config:ServiceStoreAccountConnectionString"/>
+    <add key="device.TableName" value="config:DeviceTableName"/>
+    <add key="DeviceRulesStoreContainerName" value="devicerules"/>
+    <add key="DeviceRulesTableName" value="DeviceRulesNormalizedTable"/>
+    <add key="ActionMappingStoreContainerName" value="actionmappings"/>
+    <add key="ActionMappingStoreBlobName" value="mappings.json"/>
+    <add key="AsaRefDataRulesBlobName" value="devicerules.json"/>
+    <add key="SolutionName" value="Remote Monitoring Solution"/>
+    <add key="TelemetryStoreContainerName" value="devicetelemetry"/>
+    <add key="TelemetryDataPrefix" value="devicetelemetry/"/>
+    <add key="TelemetrySummaryPrefix" value="devicetelemetry-summary/"/>
+    <add key="AlertsStoreContainerName" value="rulesoutput"/>
+    <add key="DeviceAlertsDataPrefix" value=""/>
+    <add key="MapApiQueryKey" value="config:MapApiQueryKey"/>
   </appSettings>
-
   <system.web>
```

Comment:
This section appears to be a duplicate of the `<system.web>` section on lines 15 - 18 above.